homeassistant/components/notify/xmpp.py

```diff
@@ -10,7 +10,10 @@
     ATTR_TITLE, DOMAIN, BaseNotificationService)
 from homeassistant.helpers import validate_config
 
-REQUIREMENTS = ['sleekxmpp==1.3.1', 'dnspython3==1.12.0']
+REQUIREMENTS = ['sleekxmpp==1.3.1',
+                'dnspython3==1.12.0',
+                'pyasn1==0.1.9',
+                'pyasn1-modules==0.0.8']
```

Comment:
The setup.py for sleekxmpp is wrong. The keyword for `setup` should be `install_requires` instead of `requires`: http://python-packaging.readthedocs.org/en/latest/dependencies.html#specifying-dependencies . This is what is causing us to having to explicitly write the requirements.

I think we should open a PR with sleekxmpp to fix this issue.